ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/log/gossip/DasGossipBatchLogger.java

```diff
@@ -200,14 +197,14 @@ private String msAgoString(final List<? extends Event> events) {
         + " ago";
   }
 
-  private boolean needToLogEvent(final boolean isSevereEvent) {
-    return LOG.isDebugEnabled() || (isSevereEvent && LOG.isInfoEnabled());
+  private boolean needToLogEvent() {
+    return LOG.isDebugEnabled();
   }
```

Comment:
if we're putting all to debug, why aren't we just calling log.debug?

Reply:
why not, we skip grouping computation and memory usage by events array when in info